docs/faq/component-library.md

```diff
@@ -1,9 +1,59 @@
 # Which Vue component library should I use?
 
-::: warning This answer is a stub.
-We are still working on writing the answers to the FAQ questions. The answer below is incomplete, but you may still find it useful.
-:::
+Different libraries have different strengths, so the best choice for one project isn't necessarily the best choice for another. You're going to need to judge for yourself what works best for you and your project.
 
-See <https://skirtles-code.github.io/vue-examples/#component-libraries> for a list of popular Vue 3 component libraries.
+It's unlikely that anyone on Vue Land has enough personal experience to make a detailed assessment of more than two or three libraries. It takes months of use on a real project to properly understand the pros and cons of a library, and even then those experiences won't necessarily carry across to other projects.
 
-Tailwind, <https://tailwindcss.com/>, is also a popular option if you want to write your own components but don't want to do all the CSS work yourself.
+However, if there's a specific library that you're considering, somebody might be able to share their own experiences of using it.
+
+## Assessing the options
+
+First you should consider whether you really need a component library. Would you be better off doing it yourself?
+
+Writing a component library from scratch is a lot of work, but if you don't need many components then it might be worth considering.
+
+Even if you do decide to do it yourself, it is well worth studying some existing libraries to see how they do things.
+
+It's also not an all-or-nothing decision. Some libraries will give you fully styled components that you can just use, but others provide tools to help you create your own components. For example:
+
+- [Tailwind CSS](https://tailwindcss.com/) is a popular choice for working with CSS.
+- [daisyUI](https://daisyui.com/) extends Tailwind CSS to add component-specific utility classes.
+- Libraries like [Headless UI](https://headlessui.com/) and [Radix Vue](https://www.radix-vue.com/) provide unstyled components that can be used to build your own.
+- [Tailwind UI](https://tailwindui.com/) and [shadcn-vue](https://www.shadcn-vue.com/) are large collections of example components that you can copy.
+
+If you do want a library that does it all for you, there are plenty to choose from. You'll find some suggestions listed at:
+
+- https://skirtles-code.github.io/vue-examples/component-libraries.html
+- https://ui-libs.vercel.app/
+
+Having a lot of downloads doesn't necessarily mean that a library is better than one with fewer downloads. When assessing the number of downloads you should keep in mind that newer libraries are at a massive disadvantage. Some libraries are particularly strong in a specific area, which may make them more suitable for your project than they would be for a more general audience.
```

Comment:
Wasn't sure after quick read what downloads you mean, does the library downloads something or you need to download many files to use it?
I finally figured it out, but maybe "Having a lot of downloads or github stars..." or "Being more popular..." might be a bit less confusing

Reply:
Agreed. Reworded it.